packages/rocketchat-lib/rocketchat.info

```diff
@@ -1,3 +1,4 @@
 {
-	"version": "0.69.0-develop"
+	"version": "0.69.0-develop",
+	"marketplaceApiVersion": "0.9.13"
```

Comment:
We need to get this info from the package.json file where we define the apps-engine version